earthaccess/results.py

```diff
@@ -325,9 +314,8 @@ def data_links(
                 return https_links
 
     def dataviz_links(self) -> List[str]:
-        """
-        Returns:
-            The data visualization links, usually the browse images.
+        """Returns:
+        The data visualization links, usually the browse images.
```

Comment:
The docs for this function also didn't render correctly because of a missing subject line :(

https://earthaccess--580.org.readthedocs.build/en/580/user-reference/granules/granules/#earthaccess.results.DataGranule.dataviz_links